.travis.yml

```diff
@@ -9,17 +9,17 @@ services:
 
 go:
   - 1.14
-
+ 
 go_import_path: github.com/turbonomic/kubeturbo
 
 before_install:
-  - go get -v github.com/mattn/goveralls
-
+  - go get -v github.com/mattn/goveralls@HEAD
+ 
 script:
   - make fmtcheck
   - make vet
   - make build
-  - $HOME/gopath/bin/goveralls -v -race -service=travis-ci
+  - $HOME/gopath/bin/goveralls -v -race -service=travis-ci -package='github.com/turbonomic/kubeturbo/cmd/kubeturbo github.com/turbonomic/kubeturbo/cmd/kubeturbo/app github.com/turbonomic/kubeturbo/pkg github.com/turbonomic/kubeturbo/pkg/action github.com/turbonomic/kubeturbo/pkg/action/executor github.com/turbonomic/kubeturbo/pkg/action/util github.com/turbonomic/kubeturbo/pkg/cluster github.com/turbonomic/kubeturbo/pkg/discovery github.com/turbonomic/kubeturbo/pkg/discovery/configs github.com/turbonomic/kubeturbo/pkg/discovery/detectors github.com/turbonomic/kubeturbo/pkg/discovery/dtofactory github.com/turbonomic/kubeturbo/pkg/discovery/dtofactory/property github.com/turbonomic/kubeturbo/pkg/discovery/metrics github.com/turbonomic/kubeturbo/pkg/discovery/monitoring github.com/turbonomic/kubeturbo/pkg/discovery/monitoring/kubelet github.com/turbonomic/kubeturbo/pkg/discovery/monitoring/master github.com/turbonomic/kubeturbo/pkg/discovery/monitoring/types github.com/turbonomic/kubeturbo/pkg/discovery/processor github.com/turbonomic/kubeturbo/pkg/discovery/repository github.com/turbonomic/kubeturbo/pkg/discovery/stitching github.com/turbonomic/kubeturbo/pkg/discovery/task github.com/turbonomic/kubeturbo/pkg/discovery/util github.com/turbonomic/kubeturbo/pkg/discovery/worker github.com/turbonomic/kubeturbo/pkg/discovery/worker/aggregation github.com/turbonomic/kubeturbo/pkg/discovery/worker/compliance github.com/turbonomic/kubeturbo/pkg/kubeclient github.com/turbonomic/kubeturbo/pkg/registration github.com/turbonomic/kubeturbo/pkg/resourcemapping github.com/turbonomic/kubeturbo/pkg/turbostore github.com/turbonomic/kubeturbo/pkg/util'
```

Comment:
We will need to remember to update this whitelist of packages when we add a new non-integration related package in the code. As you suggested, it would be ideal to have a command line option to allow excluding packages. Maybe add a #TODO comment?

Reply:
@ading1977 I am not sure, where to put this todo comment, except in my personal backlog :)
I think the .travis.yml might not parse ok with a comment inside. Even if it does, people adding a new package might not look at (or in some cases not even know about) this file. 
I will be pushing an update to `mattn/goveralls` soon to add an exclude-package option, and probably that will place this concern to rest. wdyt?

Reply:
That sound fine. Thanks @irfanurrehman .